### PR TITLE
implemented update and deleting users for okta as well as our database

### PR DESCRIPTION
**Pull request by reesharper**

### CODE REVIEW

**Review by c-melchor — APPROVED** [review]
Nice code, looks clean and well written. This will be a great addition to the workflow of the app.

File: data/migrations/20200625220949_create-profile.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ exports.up = (knex) => {
     .createTable('profiles', function (table) {
       table.string('id').notNullable().unique().primary();
       table.string('email').notNullable().unique();
-      table.string('name');
+      table.string('firstName');
```
Comment (ndossett) [inline data/migrations/20200625220949_create-profile.js:7]: I'm glad you had the thought to separate the name out - should be much easier for functionality in many other components as well.

**Review by ndossett — APPROVED** [review]

**Review by crystal-csete — APPROVED** [review]
This is a great fix, now the front end and back end can talk correctly.

**Review by JusticeMatthew — CHANGES_REQUESTED** [review]
Looks like this branch is a bit behind main, can you update it then resubmit your pull please and thank you!

**Review by JusticeMatthew — APPROVED** [review]

**Pull request merged**